ExampleApps/SwiftUIExample/Views/Profile/ChangePasswordView.swift

```diff
@@ -11,9 +11,12 @@ import SwiftCurrent
 
 struct ChangePasswordView: View, FlowRepresentable {
     typealias WorkflowOutput = String
-    @State private var oldPassword: String = ""
-    @State private var newPassword: String = ""
-    @State private var confirmNewPassword: String = ""
+    @State private var oldPassword = ""
```

Comment:
Minor nitpick, is there any way to create a data structure for this? A tuple, if nothing else, something so we don't have so many state variables that are just "Text for the thing, boolean for the thing"